christfellowshipchurch/apollos-api
# added a filter to prayers

### About
Added a filter that allows for only app/web submitted prayers to show up for users under the /connect tab

### Test Instructions
Open the /connect tab
Under my prayers you should only see prayers submitted by you (the user) through the web or mobile app, and you should not see prayers submitted by staff through Rock

### Closes Tickets
[CFDP-1960](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-1960)

### Related Tickets
N/A
